sdk/tables/Azure.Data.Tables/src/Azure.Data.Tables.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="$(AzureCoreSharedSources)NoBodyResponse{T}.cs" Link="Shared\Core\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="$(AzureCoreSharedSources)ForwardsClientCallsAttribute.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
```

Comment:
Nifty.... I didn't know about `%(RecursiveDir)`.   Totally stealing this.

Reply:
Cool, I totally stole it from elsewhere! 😄